addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9587,10 +9587,10 @@ msgctxt "#19071"
 msgid "Update interval"
 msgstr ""
 
-#. pvr settings "do not store epg data in local database" setting label
+#. pvr settings "store epg data in database" setting label
 #: system/settings/settings.xml
 msgctxt "#19072"
-msgid "Don't cache in local database"
+msgid "Store data in database"
```

Comment:
What about `Cache EPG data` and get rid of anything database? The user likely doesn't care where the EPG is cached, as long as it is cached in case the connection to the PVR server is slow. Just an idea, no hard feeling either way.

Reply:
I want to keep it exactly like this.